### DISCUSSION TITLE
Redshift is Deprecated and no longer maintained anymore. Use gammastep As Replacement In Future Releases

### DISCUSSION THREAD

**Discussion by Reno-Sifana** [linuxmint/discussions#174] [Ideas]
According Debian Wiki (https://wiki.debian.org/Redshift) "Redshift isn't maintained anymore and doesn't support Wayland environments. You can use [gammastep](https://packages.debian.org/gammastep), a redshift fork, as an alternative."
![Snapshot_2023-10-03_12-32-11](https://github.com/linuxmint/discussions/assets/68702583/e3ccfc8b-0fa1-4fab-b731-2e1de51f6c06)
You should replace Redshift with gammastep as Redshift replacement in Linux Mint in Future Releases (https://gitlab.com/chinstrap/gammastep)

**Comment by anaximeno** [comment]
I agree, this seems to be a nice alternative.

**Comment by rcalixte** [comment]
Both `redshift` and `gammastep` are already provided upstream by Debian/Ubuntu. Neither is installed by the user but can be added via the repositories. Also, since Mint 21.x and LMDE 5, `gammastep` has been available alongside `redshift`. One note is that `redshift` only works in X11 sessions and `gammastep` only works in sessions of wlroots-based Wayland compositors with the proper protocol support, so neither will work in Cinnamon's current Wayland implementation and `redshift` will need to continue to be used for Cinnamon's X11 session.

**Reply by anaximeno** [reply]
Yeah, this makes sense, so probably another solution should be found long term.

**Comment by claudiux** [comment]
Ping @raphaelquintao

**Comment by Jopp-gh** [comment]
Wayland is still in development, so right now Redshift seems to be ok as solution. 

Problem, Redshift needs coordinates to work and that's bad for privacy. 

Redshift's wiki suggests that earlier versions had a  clock to start the night screen.

Anyway, Redshift is a bit old and there are definitely better alternatives available, like
https://cinnamon-spices.linuxmint.com/applets/view/313

Problem is, I want a pre-installed and integrated  app. Some Applets can crash cinnamon and I've seen cinnamon crashing multiple times bc of plugins and applets.

**Comment by u2fly** [comment]
> One note is that `redshift` only works in X11 sessions and `gammastep` only works in sessions of wlroots-based Wayland compositors with the proper protocol support, so neither will work in Cinnamon's current Wayland implementation and `redshift` will need to continue to be used for Cinnamon's X11 session.

Meanwhile, there is `xsec` for X11:

- https://github.com/faf0/sct
  - https://packages.debian.org/sid/xsct
